code/xcvm/cosmwasm/tests/src/tests/suite.rs

```diff
@@ -283,12 +283,25 @@ fn to_canonical(account: Account) -> CanonicalAddr {
 }
 
 fn load_contracts() -> XCVMContracts {
-	let code_cw20 = std::fs::read(std::env::var("CW20").unwrap()).unwrap();
 	let code_asset_registry =
 		std::fs::read(std::env::var("CW_XCVM_ASSET_REGISTRY").unwrap()).unwrap();
 	let code_interpreter = std::fs::read(std::env::var("CW_XCVM_INTERPRETER").unwrap()).unwrap();
 	let code_router = std::fs::read(std::env::var("CW_XCVM_ROUTER").unwrap()).unwrap();
 	let code_gateway = std::fs::read(std::env::var("CW_XCVM_GATEWAY").unwrap()).unwrap();
+
+	// When running the test locally, the cw20_base.wasm file is downloaded by
+	// the Build script (see build.rs).  However, on CI when running under NIX
+	// downloading is disabled and to run those tests CW20 environment variable
+	// needs to be set and point at cw20_base.wasm file.
+	let pathbuf;
+	let path = if let Some(var) = std::env::var_os("CW20") {
```

Comment:
who sets the env var cw20?

Reply:
It needs to be set by whoever runs the test.  Since this is only meant for CI it, if we want to run those tests in CI we’d need to download the contract and set the variable in CI.